source/BoardingPanel.cpp

```diff
@@ -637,21 +654,24 @@ bool BoardingPanel::Plunder::CanTake(const Ship &ship) const
 void BoardingPanel::Plunder::Take(int count)
 {
 	this->count -= count;
-	UpdateStrings();
+	UpdateStrings(false);
```

Comment:
```suggestion
	UpdateStrings();
```
with it defaulting on false